src/directives/leaflet.js

```diff
@@ -24,9 +23,9 @@ angular.module("leaflet-directive", []).directive('leaflet',
         transclude: true,
         template: '<div class="angular-leaflet-map"><div ng-transclude></div></div>',
         controller: function ($scope) {
-            _leafletMap = $q.defer();
+            this._leafletMap = $q.defer();
```

Comment:
Yeap but since the _leafletMap is outside the link function, there is a risk that it might be overwrite when the second Map is linked.

I don't know how Angular handles the cleanup, I just thought that the attributes of the controller will be destroyed if the directive is destroyed.

BTW, what is wrong with the build? I have noticed that all the builds fail since d386b7f. Is it due to a new version of Chrome?